app/components/conversations/conversation-part-reopened.js

```diff
@@ -0,0 +1,22 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed, get } from '@ember/object';
+import { alias } from '@ember/object/computed';
+
+export default Component.extend({
+
+  author: null,
+
+  classNames: ['conversation-part', 'conversation-part--reopened'],
+  classNameBindings: ['isSelf:conversation-part-reopened--is-self'],
+
+  reopenedAt: null,
+
+  currentUser: service(),
+
+  isSelf: computed('author', 'user', function() {
+    return get(this, 'author.id') === get(this, 'user.id');
+  }),
+
+  user: alias('currentUser.user')
```

Comment:
There are some issues here that are at odds with the style guide. See https://github.com/code-corps/code-corps-ember/blob/develop/docs/STYLEGUIDE.md#ember-modules